src/styles/index.scss

```diff
@@ -1,3 +1,70 @@
-body {
-  margin: 0;
+$calendar-week-days-sum: 7;
+$calendar-days: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+$start-day: 'sun';
+$calendar-month-length: 31;
+$general-font-family: Arial, Helvetica, sans-serif;
+$general-font-size: 30px;
+$calendar-size: 100px;
+$calendar-padding: 10px;
+
+html {
+  font-family: $general-font-family;
+  font-size: $general-font-size;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.calendar {
```

Comment:
According to BEM rules each BEM-block should be in a separate file

Also, you can move variables to a separate file